apps/src/clientApi.js

```diff
@@ -13,6 +13,10 @@ function clientApi(endpoint) {
       return '/v3/' + endpoint + '/' + dashboard.project.getCurrentId() + (path ? '/' + path : '');
     },
     ajax: function (method, file, success, error, data) {
+      if (!window.dashboard) {
+        error({status: "No dashboard"});
+        return;
+      }
```

Comment:
Did you run into this? i.e. is there a particular scenario where we expect to hit it?